src/providers/ffz/FfzEmotes.cpp

```diff
@@ -229,6 +229,12 @@ void FfzEmotes::loadChannel(
                     shared->addMessage(makeSystemMessage(
                         "This channel has no FrankerFaceZ channel emotes."));
             }
+            else if (result.status() == 404)
+            {
+                // FFZ emotes not found
+                shared->addMessage(makeSystemMessage(
```

Comment:
404 is an "expected error" in case the channel does not have any emotes. Checking for the error like you do is great, but I don't think we need to tell the users that no emotes are found.